package.json

```diff
@@ -11,7 +11,8 @@
     "lab": "5.x.x"
   },
   "scripts": {
-    "test": "make test-cov",
+    "test": "node node_modules/lab/bin/lab -a code -t 100 -L",
```

Comment:
IIRC npm installs a batch script that shims the bin to work in windows, so just pointing directly to lab should work just fine